src/main/scala/net/snowflake/spark/snowflake/Utils.scala

```diff
@@ -49,7 +49,12 @@ object Utils {
     */
   val SNOWFLAKE_SOURCE_NAME = "net.snowflake.spark.snowflake"
 
-  val VERSION = "2.5.6"
+  /**
+    * Short literal name of SNOWFLAKE_SOURCE_NAME
+    */
+  val SNOWFLAKE_SOURCE_SHORT_NAME = "snowflake"
+
+  val VERSION = "2.5.8"
```

Comment:
Did you change this version name in last release?

Reply:
No. It was 2.5.6